src/java/org/apache/cassandra/exceptions/RequestFailureReason.java

```diff
@@ -37,7 +37,8 @@
     TIMEOUT                  (2),
     INCOMPATIBLE_SCHEMA      (3),
     READ_SIZE                (4),
-    NODE_DOWN                (5);
+    NODE_DOWN                (5),
+    INDEX_NOT_AVAILABLE      (6);
```

Comment:
I guess this isn't strictly a protocol change, in the sense that it changes a format. It would be a new code, but so was the CASSANDRA-17164 addition of `NODE_DOWN`. I suppose clients would still have to adjust to properly interpret it.